frontend/public/components/monitoring/dashboards/index.tsx

```diff
@@ -733,7 +739,7 @@ type Row = {
   title?: string;
 };
 
-type Board = {
+export type Board = {
```

Comment:
It would be good to move `Board` to `public/components/monitoring/types.ts` if we're using it in more than one file.